### ISSUE TITLE
Resell

### ISSUE THREAD

**Issue by Haunterrr** [Maxx53/SCMBot#425]
Авт. продажа купленного работает?
Или я где то плуганул и что то не настроил, но уже больше недели бот скупает шмот и ждет вмешательства ручек)))

**Comment by fallenQQ** [comment]
да, чтот не пашет перепродажа . сам вот сегодня заметил, когда добавил новые слоты , а старые слоты нормально перепродает
